src/types/formData.ts

```diff
@@ -0,0 +1,33 @@
+// Copied over from form-data.
+import * as http from "http";
+import * as stream from "stream";
+
+export interface IFormData extends stream.Readable {
```

Comment:
I'm strongly against the Hungarian notation in general, but especially for interface names in TypeScript.

So are [TypeScript's own guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names) (note: not general guidelines, as stated in the page).

[TSLint also has that rule by default](https://palantir.github.io/tslint/rules/interface-name), but typescript-eslint doesn't anymore after a [long discussion about it](https://github.com/typescript-eslint/typescript-eslint/issues/374).

Anyway, this is subjective and personal, so if you and the team prefer that, it's fine.

Reply:
This is the way this project was configured before and I didn't want to change too much with a simple change like this. Thanks for the recommendation!